src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -252,13 +252,13 @@ public static String formatDateTimeForInstructorHomePage(Date date) {
         return sdf.format(date);
     }
 
-    public static String formatDateForFeedbackSubmissionEditPage(Date date, double timeZone) {
+    public static String formateDateToIso8601(Date date, double timeZone) {
```

Comment:
There's a typo in the method name. Also, move the conversion to UTC logic into a separate method, as from the method name, this method should do nothing other than conversion.